tools/devtools/src/devtools_app/devtools_window/events_tab/events_tab.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<templates xml:space="preserve">
+  <t t-name="devtools.EventsTab" owl="1">
+    <div class="panel-top">
+      <i title="Start/Stop Recording" class="fa fa-circle pointer-icon pe-2" t-attf-style="color: {{store.activeRecorder ? 'rgb(255, 31, 31)' : 'rgb(0, 0, 0)'}};" t-on-click.stop="toggleRecording" aria-hidden="true"></i>
+      <div class="icons-separator"/>
+      <i title="Clear events" class="fa fa-ban lg-icon pointer-icon px-2" t-on-click.stop="clearConsole" aria-hidden="true"></i>
+      <div class="icons-separator"/>
+      <i title="Show events as tree" class="fa fa-tree lg-icon pointer-icon px-2" t-attf-style="color: {{store.eventsTreeView ? 'rgb(41, 134, 255)' : 'rgb(0, 0, 0)'}};" t-on-click.stop="toggleEventsAsTree" aria-hidden="true"></i>
+      <div class="icons-separator"/>
+      <!-- <EventSearchBar/> -->
+    </div>
+    <div class="horizontal-border"/>
+    <div class="events-container">
+      <t t-if="showHelp()">
+        <div class="status-message">
+          Click on the <i class="fa fa-circle px-2" style="font-size: 0.8em;"></i> button to start recording events.
+        </div>
+      </t>
+      <t t-else="">
+        <t t-if="store.eventsTreeView">
+          <t t-foreach="store.eventsTree" t-as="event" t-key="event.id">
+            <EventNode t-props="event"/>
+          </t>
+        </t>
+        <t t-else="">
+          <t t-foreach="store.events" t-as="event" t-key="event_index">
+            <Event t-props="event"/>
+          </t>
+        </t>
+        <div class="blank-space"/>
```

Comment:
It is just to leave a space under the events so that it is simpler to read when scrolling
![image](https://user-images.githubusercontent.com/56196503/221215046-aae457a5-417c-4520-9642-fc5f54f0b4d3.png)